tests/validator/test_validator.py

```diff
@@ -1007,45 +1007,53 @@ def test_validator_include_rendered_content(
         )
     )
 
-    expected_expectation_validation_result_rendered_content = RenderedAtomicContent(
-        name="atomic.diagnostic.observed_value",
-        value=RenderedAtomicValue(
-            schema={"type": "com.superconductive.rendered.string"},
-            params={},
-            template="6",
-        ),
-        value_type="StringValueType",
+    expected_expectation_validation_result_diagnostic_rendered_content = (
```

Comment:
Just curious why you added `diagnostic` to the name, to me a `validation_result` is implicitly a diagnostic-type expectation. It feels a bit repetitive.

Reply:
I think that is correct, but I feel that we need to be precise about these "modes", or we could start to get confused in the future. The `ExpectationConfiguration` in the `ExpectationValidationResult` does have both a "prescriptive" and "diagnostic" mode due to the difference in the way we render the evaluation parameters, but that this not what we are testing on this line.